src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandStatus.java

```diff
@@ -3,6 +3,8 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 public enum CommandStatus {
```

Comment:
I know it was already missing, but a brief Javadoc would be useful here?

Reply:
Added documentation